Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -118,4 +81,32 @@ class WelcomeViewController: BaseViewController {
         }
     }
     
+    fileprivate func processEnvironmentURL(newEnvironmentURL:String, currentEnvironmentURL:String) {
+        EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+        self.didEditTextField = false
+        if currentEnvironmentURL != newEnvironmentURL{
+            ActivitiesRequestManager.sharedInstance.getActivityDefaultCategories(onCompletion: { (success, message, code, activitiesReturned, error) in
+                if success { EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+                }else{ EnvironmentManager.updateEnvironment(withURLString: currentEnvironmentURL)
+                }
+            })
+        }else{
+            if (self.didEditTextField && currentEnvironmentURL != newEnvironmentURL){
```

Comment:
Make this an ``else if`` statement by putting the ``if`` straight after the ``else``.

Reply:
Modified accordingly.